cmake/ctest/drivers/atdm/ats1/drivers/Trilinos-atdm-ats1-knl_intel-19.0.4_mpich-7.7.6_openmp_static_opt.sh

```diff
@@ -2,5 +2,5 @@
 if [ "${Trilinos_TRACK}" == "" ] ; then
   export Trilinos_TRACK=Experimental
 fi
-export SBATCH_TIME_LIMIT_MINUTES=900 # 15 hour time limit
+export SBATCH_TIME_LIMIT_MINUTES=1440 # 24 hr time limit
```

Comment:
We can't actually run for a full 24 hours.  Otherwise, this might bleed into the next testing day.

Can we set a more reasonable time limit?  If we are building correctly, I don't think the the build should take over a few hours.  Even when building on the single shared hammered login node 'mutrino' using just `ninja -j8`, I don't think we ever saw build times over 9 hours.  We should be building on a HSW compute node with more parallelism in much faster than that.